flow/util/genMetrics.py

```diff
@@ -404,7 +413,20 @@ def extract_metrics(cwd, platform, design, flow_variant, output, hier_json):
                     continue
                 print(f"Extract Metrics for {plt}, {des}, {variant}")
                 file = "/".join(["reports", plt, des, variant, "metrics.json"])
-                metrics, df = extract_metrics(cwd, plt, des, variant, file, args.hier)
+                logPath = os.path.join(cwd, "logs", plt, des, variant)
+                rptPath = os.path.join(cwd, "reports", plt, des, variant)
+                resultPath = os.path.join(cwd, "results", plt, des, variant)
```

Comment:
Why are you recomputing these here?  I thought the point is to pass them in.

Reply:
This is the non-utils.mk use case where the .py file scans multiple projects. Is it used? I wanted to let that dog lie in this PR.

Reply:
I think its is used for a bulk update but not certain.  I'm glad to leave it alone.